ZCRMiOS/Classes/DashBoardAPIHandler.swift

```diff
@@ -0,0 +1,1737 @@
+//
+//  DashBoardAPIHandler.swift
+//
+//  Created by Kalyani shiva on 12/07/18.
+//
+
+
+import Foundation
+
+
+class DashBoardAPIHandler: CommonAPIHandler {
+    
+    // Avoids ambiguity of back and front slashes in path names like  /\(analytics)/\(dbID)
+    let SLASH = "/"
```

Comment:
not needed this kind of ambiguity issues will be arrested in our test cases.

Reply:
I will remove the SLASH but HYPHEN and NONE are constants used by a function called 'convertLabelToNoneIfHyphen' . They are not used to avoid ambiguity .